assets/xml/objects/object_zl4.xml

```diff
@@ -1,125 +1,177 @@
-        <Texture Name="object_zl4_Tex_001DF0" OutName="tex_001DF0" Format="ci8" Width="8" Height="8" Offset="0x1DF0" />
-        <Texture Name="object_zl4_Tex_001E30" OutName="tex_001E30" Format="rgba16" Width="16" Height="32" Offset="0x1E30" />
-        <Texture Name="object_zl4_Tex_002230" OutName="tex_002230" Format="ci8" Width="8" Height="8" Offset="0x2230" />
-        <Texture Name="object_zl4_Tex_002270" OutName="tex_002270" Format="ci8" Width="8" Height="16" Offset="0x2270" />
-        <Texture Name="object_zl4_Tex_0022F0" OutName="tex_0022F0" Format="ci8" Width="16" Height="32" Offset="0x22F0" />
-        <Texture Name="object_zl4_Tex_0024F0" OutName="tex_0024F0" Format="rgba16" Width="16" Height="16" Offset="0x24F0" />
-        <Texture Name="object_zl4_Tex_0026F0" OutName="tex_0026F0" Format="rgba16" Width="16" Height="16" Offset="0x26F0" />
-        <Texture Name="object_zl4_Tex_0028F0" OutName="tex_0028F0" Format="rgba16" Width="8" Height="8" Offset="0x28F0" />
-        <Texture Name="object_zl4_Tex_002970" OutName="tex_002970" Format="rgba16" Width="8" Height="8" Offset="0x2970" />
-        <Texture Name="object_zl4_Tex_0029F0" OutName="tex_0029F0" Format="rgba16" Width="16" Height="8" Offset="0x29F0" />
-        <!-- <Blob Name="object_zl4_Blob_002AF0" Size="0x2C00" Offset="0x2AF0" /> -->
-        <Texture Name="object_zl4_Tex_0056F0" OutName="tex_0056F0" Format="rgba16" Width="16" Height="16" Offset="0x56F0" />
-        <Texture Name="object_zl4_Tex_0058F0" OutName="tex_0058F0" Format="ci8" Width="16" Height="16" Offset="0x58F0" />
-        <Texture Name="object_zl4_Tex_0059F0" OutName="tex_0059F0" Format="rgba16" Width="8" Height="8" Offset="0x59F0" />
-        <Texture Name="object_zl4_Tex_005A70" OutName="tex_005A70" Format="rgba16" Width="16" Height="16" Offset="0x5A70" />
-        <Texture Name="object_zl4_Tex_005C70" OutName="tex_005C70" Format="ci8" Width="8" Height="8" Offset="0x5C70" />
-        <Texture Name="object_zl4_Tex_005CB0" OutName="tex_005CB0" Format="ci8" Width="16" Height="16" Offset="0x5CB0" />
-        <Texture Name="object_zl4_Tex_005DB0" OutName="tex_005DB0" Format="rgba16" Width="32" Height="32" Offset="0x5DB0" />
-        <DList Name="object_zl4_DL_00A630" Offset="0xA630" />
-        <DList Name="object_zl4_DL_00AFC8" Offset="0xAFC8" />
-        <DList Name="object_zl4_DL_00B8D8" Offset="0xB8D8" />
-        <DList Name="object_zl4_DL_00BB40" Offset="0xBB40" />
-        <DList Name="object_zl4_DL_00BC90" Offset="0xBC90" />
-        <DList Name="object_zl4_DL_00BE70" Offset="0xBE70" />
-        <DList Name="object_zl4_DL_00C0E0" Offset="0xC0E0" />
-        <DList Name="object_zl4_DL_00C230" Offset="0xC230" />
+        <!-- Child Zelda: from the cutscene where Deku Link recalls how to play Song of Time after getting his Ocarina back -->
+
+        <Animation Name="gDmZl4IdleAnim" Offset="0x654" />
```

Comment:
What do you think about prefixing these with `gChildZelda` instead of `gDmZl4`? I tend to prefix objects with what it actually *is* rather than the filename.

Reply:
I'd rather have `ChildZelda` myself.